packages/paste-core/core-bundle/absolute/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "@twilio-paste-core/absolute",
+  "version": "0.0.0",
+  "private": true,
+  "sideEffects": false,
+  "main": "../dist/absolute.js",
+  "types": "../dist/absolute.d.ts"
+}
```

Comment:
This file shouldn't be committed. We removed the component, but it might be a folder that's hanging about on your machine